h2/src/main/org/h2/table/InformationSchemaTable.java

```diff
@@ -1849,6 +1863,23 @@ && checkIndex(session, constraint.getName(), indexFrom, indexTo))
                         (ConstraintReferential) constraint));
     }
 
+    private void restorePoints(SessionLocal session, Value indexFrom, Value indexTo, ArrayList<Row> rows) {
```

Comment:
Is it safe to expose this information to non-admin users?

Reply:
H2 hides many other objects not accessible for regular users, so I think it should hide this information too.